infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -417,6 +417,7 @@ function staging_special_case__k8s_staging_kustomize() {
     principal="group:${owners}"
     ensure_project_role_binding "${project}" "${principal}" "roles/cloudkms.admin"
     ensure_project_role_binding "${project}" "${principal}" "roles/cloudkms.cryptoKeyEncrypter"
+    ensure_project_role_binding "${project}" "${principal}" "roles/ondemandscanning.admin"
```

Comment:
Suggest having a comment above to explain why we need this.

Also, putting this line here will give this permission to the kustomize owners (e.g. me, Katrina, and Jeff). The PR title makes it sound like we actually want to give it to the cloud build service account? In that case, we should move this line to below 426 at the end of this function.

Reply:
Thank you. Fixed